dotnet/Firely.Server.MessageSender/appsettings.json

```diff
@@ -4,5 +4,18 @@
       "Default": "Information",
       "Microsoft.Hosting.Lifetime": "Information"
     }
+  },
+  "ImportOptions": {
```

Comment:
Having some part interactive (having to type `dir`) and some part taken from the config file (the folder name) is a bit confusing. I would suggest that either we offer a fully automated process in addition to interactive process, or we keep all params interactive.

Reply:
I agree. The advantage of keeping this interactive is that you can now send multiple requests asynchronously, and pause them in RabbitMQ or Firely Server, to demonstrate how the message queue processing can "catch up". So it's a bit unusual, maybe only useful for demonstration.

Reply:
I moved the "import" mode to non-interactive